yt_dlp/extractor/facebook.py

```diff
@@ -445,6 +445,9 @@ def _perform_login(self, username, password):
         try:
             login_results = self._download_webpage(request, None,
                                                    note='Logging in', errnote='unable to fetch login page')
+            if "Your Request Couldn" in login_results:
+                raise ExtractorError('Failed to perform login request.')
+
             if re.search(r'<form(.*)name="login"(.*)</form>', login_results) is not None:
```

Comment:
```suggestion
            if 'Your Request Couldn' in login_results:
                self.raise_login_required('Failed to login with credentials', method='cookies')
            elif re.search(r'<form(.*)name="login"(.*)</form>', login_results) is not None:
```

Reply:
IMO this error suggests that there is something wrong with what user provided rather than program itself. Do you know if anyone successfully logged in here in a while using user/pass?

Reply:
With this we would be telling them to use cookies instead. Isn't that what we want?

Reply:
That's true. But like I said. It leaves impression that it's something that user did wrong and that caused user/pass parameters to not work. And they might try 5x times to try to get this work (typo in pass etc) - still failing, rather than focus on `preffered` way of just using cookies.

Reply:
AFAIK FB only accept `encpass` in form currently, which this method doesn't support. Hence I'm pretty convinced it doesn't work at all.